dataclients/routestring/string.go

```diff
@@ -0,0 +1,31 @@
+// Package routestring provides a DataClient implementation for
+// setting route configuration in form of simple eskip string.
```

Comment:
an example would be nice :)

Reply:
i'd like to add an example to this one using the inlineContent() filter from this PR: https://github.com/zalando/skipper/pull/480

so please let's have a look at the other one first

Reply:
done